luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -4,8 +4,10 @@
 
 import com.yahoo.bard.webservice.data.config.ConfigurationLoader;
 import com.yahoo.bard.webservice.data.config.ResourceDictionaries;
-import com.yahoo.bard.webservice.data.dimension.Dimension;
-import com.yahoo.bard.webservice.data.dimension.DimensionDictionary;
+import com.yahoo.bard.webservice.data.dimension.*;
```

Comment:
We don't use star imports in Fili. All imports should be explicitly imported. To keep IntelliJ from automatically collapsing your imports into stars: https://stackoverflow.com/questions/3587071/disable-intellij-starred-package-imports